tools/adios2pio-nm/adios2pio-nm-lib.cxx

```diff
@@ -1575,7 +1573,7 @@ int ConvertVariableTimedPutVar(IOVector &bpIO, EngineVector &bpReader,
 }
 
 template <class T>
-int adios2_ConvertVariableDarray(adios2::Variable<T> *v_base, std::vector<T> v_value,
+int adios2_ConvertVariableDarray(adios2::Variable<T> *v_base, std::vector<T> &v_value,
```

Comment:
Is v_base and v_value used outside of this function (Is it required to pass it as an arg)?

Reply:
@tkurc Could you please take a look?

Reply:
We don't need to pass them as arguments. They are left over from initial coding and debugging.

Reply:
Ok, we will need to refactor the code in a future PR.